src/components/common/ColTagList.js

```diff
@@ -14,17 +14,23 @@ function ColTagList(props) {
         className="innerTag"
         data-testid="doc-tag"
         onClick={() => {
-          searchDocs(tag, authState);
+          searchDocs(tag, authState, 1, pageSize);
+          setCurrentSearch(tag, 1, pageSize);
         }}
       >
-        {tag.length < 45 ? (
+        {tag.length < 36 ? (
```

Comment:
This change wasn't covered in the video... is this part of the bug fix? It looks like you are slicing the tags to get rid of the overhang? Just curious how this looks on the page.

Reply:
I was actually playing with the code and i guess i forgot to change that part back, that is a good catch , i'll fix it right away. Thanks